packages/components/stories/GuidedTour.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import { I18nextProvider, translate } from 'react-i18next';
+import i18n, { LanguageSwitcher } from './config/i18n';
+
+import { GuidedTour } from '../src/index';
+import I18N_DOMAIN_COMPONENTS from '../src/constants';
+
+class ImportDemo extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			loading: false,
+			finish: false,
+		};
+		this.onClick = this.onClick.bind(this);
```

Comment:
Not required at all but you can use the transform-class-properties to autobind the function instead of make this bind in a constructor ( same for the state ) like :

```
class ImportDemo extends React.Component {
    state = {
        loading: false,
        finish : false,
    }

    onClick = () => {
         this.props.beforeLoading();
         ......
    }

}
```

Reply:
It will be lighter 👍